contrib/babelfishpg_tsql/src/bbf_parallel_query.c

```diff
@@ -105,23 +117,29 @@ bbf_ExecInitParallelPlan(EState *estate, ParallelContext *pcxt, bool estimate)
 									get_relkind_objtype(get_rel_relkind(perminfo->relid)),
 									get_rel_name(perminfo->relid));
 				}
-				temp_relids_local = bms_add_member(temp_relids_local, rte->relid);
+				if (rte->relid <= (Oid) INT32_MAX)
+					temp_relids_local = bms_add_member(temp_relids_local, rte->relid);
+				else
+					temp_relids_beyond_int32_local = bms_add_member(temp_relids_beyond_int32_local, (rte->relid - (Oid) INT32_MAX - 1U));
 			}
 		}
 		temp_relids_str = bmsToString(temp_relids_local);
+		temp_relids_beyond_int32_str = bmsToString(temp_relids_beyond_int32_local);
 
```

Comment:
what if temp_relids_beyond_int32_local is NULL? what will bmsTostring return?

Reply:
it will be empty representation of bms like `(b)`